blueprints/app/files/tests/helpers/index.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  setupApplicationTest as upstreamSetupApplicationTest,
+  setupRenderingTest as upstreamSetupRenderingTest,
+  setupTest as upstreamSetupTest,
+} from 'ember-qunit';
+
+// This file exists to provide wrappers around ember-qunit's / ember-mocha's
+// test setup functions. This way, you can easily extend the setup that is
+// needed per test type.
+
+function setupApplicationTest(
+  hooks: Parameters<typeof upstreamSetupApplicationTest>[0],
+  options?: Parameters<typeof upstreamSetupApplicationTest>[1]
```

Comment:
TS team: please have a look here!

I could have typed the first arg as `hooks: NestedHooks`, as `@types/ember-qunit` declares this type globally. But not the `SetupOptions` interface, which is not even exported. So the only way I found to type this was like this. And for consistency I chose to do this for both args...

Reply:
This could also be `(...[hooks, options]: Parameters<typeof upstreamSetupApplicationTest>)`, not that that's _much_ better. I think we should probably just export the `SetupOptions` type from `@types/ember-qunit`.

Reply:
I am doing that now!

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62073 – I exported it as `@internal`, since it is expressly *not* part of the public API, but *is* important for collaborators here. I also opened https://github.com/emberjs/ember-qunit/issues/957 to track converting `ember-qunit` to TS!

Reply:
Using the new export now!